Robot2019/src/main/java/frc/robot/lib/Limelight.java

```diff
@@ -0,0 +1,71 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.lib;
+
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class Limelight {
+  public enum Mode {
+    DIST, STEER, TARGET
+  }
+
+  private NetworkTableInstance inst;
+  private NetworkTable table;
+  private double tv, tx, ty, ta;
+
+  public Limelight() {
+    inst = NetworkTableInstance.getDefault();
+    table = inst.getTable("limelight");
+  }
+
+  // Adjusts the distance between a vision target and the robot. Uses basic PID with the ty value from the network table.
+  public double distanceAssist() {
+    tv = table.getEntry("tv").getDouble(0.0);
+    ty = table.getEntry("ty").getDouble(0.0);
```

Comment:
Please document what tv, ty, and ta are and, if appropriate, include a link to relevant limelight doc.